### ISSUE TITLE
New ideas for upgrading

### ISSUE THREAD

**Issue by Alireza-F-kenedy** [bepass-org/oblivion-desktop#323]
### Feature description | توضیحات ویژگی

Hello Dear Mark,
I have an idea to enhance the permanent scanning feature of the endpoint application. When an endpoint is connected to a manually entered IP address in the "Manually Entered IPs" section of the scanner, the application should save the configuration and assign a name to it. This will be particularly useful when using the application on different networks, such as mobile data and home Wi-Fi, which have different IP addresses.

In other words, in the endpoint section, the IP is manually entered and saved, because each IP works differently for each network, and to use the program for those who have two different Internets (mobile Internet and home Internet), changing some settings It gets boring

In the endpoint section, save three options (default) (suggested one) (suggested two) in this profile section, with the name, for example, house or mobile, or any desired name.

**Comment by Alireza-F-kenedy** [comment]
I wrote a mistake in the Android section

**Comment by ircfspace** [comment]
سلام
ضمن تشکر از پیشنهاد شما،
قابلیتی به صورت آزمایشی به برنامه تحت عنوان <پروفایل> اضافه‌شد، تا بتونین اندپوینت‌های موردنظر رو با عناوین دلخواه ذخیره و مورد استفاده قرار بدین
این‌مورد در بروزرسانی آتی در دسترس قرار میگیره
